--- .oh-my-zsh/themes/agnoster.zsh-theme.orig	2022-08-25 16:16:35.125020000 +0200
+++ .oh-my-zsh/themes/agnoster.zsh-theme	2022-08-25 16:34:26.071072000 +0200
@@ -242,6 +242,13 @@
   esac
 }
 
+prompt_kubecontext() {
+kubectl config current-context &> /dev/null
+if [ $? -eq 0 ]; then
+  prompt_segment green black "(`kubectl config current-context`)"
+fi
+}
+
 ## Main prompt
 build_prompt() {
   RETVAL=$?
@@ -249,6 +256,7 @@
   prompt_virtualenv
   prompt_aws
   prompt_context
+  prompt_kubecontext
   prompt_dir
   prompt_git
   prompt_bzr
